contracts/math/andromeda-point/src/testing/tests.rs

```diff
@@ -14,83 +14,40 @@ fn test_instantiation() {
 #[test]
 fn test_set_and_update_point() {
     let (mut deps, info) = proper_initialization(PointRestriction::Private);
-    let point = PointCoordinate::from_f64(10_f64, 10_f64, Some(10_f64));
-    point.validate().unwrap();
+    let point = PointCoordinate {
+        x_coordinate: SignedDecimal::percent(1000),
```

Comment:
Any reason we don't use `from_ratio`? I've noticed this is a common occurence, if the issue is the readability of something like `from_ratio` we should potentially create a wrapper function to help generate these `SignedDecimal` values.